code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -221,6 +227,13 @@
 			playsound(loc, 'sound/effects/voxrustle.ogg', 50, 0) //Credit to sound-ideas (freesfx.co.uk) for the sound.
 			m_type = 2
 
+		if("warble", "warbles")
+			var/M = handle_emote_param(param)
+
+			message = "<B>[src]</B> warbles[M ? " at [M]" : ""]."
+			playsound(loc, 'sound/effects/warble.ogg', 50, 0) //Gredit to alienistcog (freesound.org) for the sound.
```

Comment:
This is probably a more license compliant way to cite the person. Also, "Credit" was mispelled
```suggestion
			playsound(loc, 'sound/effects/warble.ogg', 50, 0) // Copyright CC BY 3.0 alienistcog (freesound.org)
```